control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -255,6 +202,61 @@ pub(super) async fn main() -> anyhow::Result<()> {
                         .help("Uuid of the volume to unfreeze")
                 )
         )
+        .subcommand(
```

Comment:
Do we still need the subcommands? Or can we just use the code directly with spawn_blocking?

Reply:
Yeah yeah we can now since we have decided to live with the fact the manual kill of the pod is the way out. I already have the changes on the other PR. Both of them work. Please review the other one if you think we should do that one. cc @dsharma-dc